### PR TITLE
Keyboard Shortcuts / Hotkeys

### PR DESCRIPTION
**Pull request by PaulHax**
Default key mappings in `config.ts`.  Can change keys in config.JSON.   

Related #390

### CODE REVIEW

File: src/composables/useKeyboardShortcuts.ts [hunk 1]
```diff
@@ -1,34 +1,25 @@
...
-  const activeToolStore = toolToStore[toolStore.currentTool];
-  if (!activeToolStore) return;
-
-  const labels = Object.entries(activeToolStore.labels);
-  const activeLabelIndex = labels.findIndex(
-    ([name]) => name === activeToolStore.activeLabel
-  );
+export function useKeyboardShortcuts() {
+  const keys = useMagicKeys();
+  const unwatchFuncs = ref([] as Array<ReturnType<typeof whenever>>);
```
Comment (floryst) [inline src/composables/useKeyboardShortcuts.ts:12]: `unwatchFuncs` has no reason to be a ref.

Comment (PaulHax) [reply src/composables/useKeyboardShortcuts.ts]: Yes!  Fixed.

File: src/composables/useKeyboardShortcuts.ts [hunk 2]
```diff
@@ -1,34 +1,25 @@
...
+import { ACTION_TO_FUNC } from './actions';
 
-const applyLabelOffset = (offset: number) => {
-  const toolToStore = {
-    [Tools.Rectangle]: useRectangleStore(),
-    [Tools.Ruler]: useRulerStore(),
-    [Tools.Polygon]: usePolygonStore(),
-  };
-  const toolStore = useToolStore();
+export const actionToKey = ref(ACTION_TO_KEY);
```
[the remark this answers is not in the document]
Comment (PaulHax) [reply src/composables/useKeyboardShortcuts.ts:8]: That would be nice.  Tackel whenever we get the "configure shortcuts" GUI?  At the moment, can only configure shortcuts with config.ts or config.JSON.

**Comment by floryst** [comment]
LGTM

**Pull request merged**